### PR TITLE
actually stashing gold (wasnt implemented yet)

### PR DESCRIPTION
**Pull request by Cr4zyEight**
guess commit message is enough :)

### CODE REVIEW

File: src/ui_manager.py [hunk 1]
```diff
@@ -10,9 +10,9 @@
 import os
 import numpy as np
 from logger import Logger
-from utils.misc import wait, cut_roi
+from utils.misc import wait, cut_roi, color_filter, send_discord
 from config import Config
-from utils.misc import color_filter
+import threading
```
Comment (aeon0) [inline src/ui_manager.py:15]: why import threading here?

Comment (aeon0) [inline src/ui_manager.py:15]:
```suggestion
```

Comment (Cr4zyEight) [reply src/ui_manager.py:15]: That was for my custom discord hook to see if hes actually stashing :) just forgot to delete the import

**Comment by Cr4zyEight** [comment]
deleted the unused import

**Pull request merged**